frappe/__init__.py

```diff
@@ -1364,7 +1364,12 @@ def get_active_domains():
 
 	return active_domains
 
-def get_system_country():
-	if local.system_country is None:
-		local.system_country = db.get_single_value('System Settings', 'country') or ''
-	return local.system_country
+def get_system_settings():
+	if not local.system_settings:
+		local.system_settings = _dict()
+	if "system_country" not in local.system_settings:
```

Comment:
why this???

This must be passed as a parameter, no?

`def get_system_settings(key):`